test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -1493,7 +1493,7 @@ static function (ReflectionClass $interface) : string {
         sort($dInterfaces);
 
         self::assertSame(['B'], $cInterfaces);
-        self::assertSame(['A', 'B', 'C'], $dInterfaces);
+        self::assertSame(['A', 'C'], $dInterfaces);
```

Comment:
Yes, this was bug: https://github.com/Roave/BetterReflection/blob/master/test/unit/Fixture/ClassWithInterfaces.php#L45

Reply:
Good catch! :muscle: